### ISSUE TITLE
Is a Raspberry PI too weak? Crontab settings possible?

### ISSUE THREAD

**Issue by ingeniumdesign** [towfiqi/serpbear#253]
I have 7 domains and 120 keywords (7 Domains | 120 keywords) running on my Raspberry PI with Docker.  

But it doesn't manage to update the 120 keywords once a day?  
For many keywords in the ‘Position’ column there is always the loading symbol.   
Is this due to the performance of Raspberry PI or something else?  
  
Is it possible to set the scan time and intervals per domain?  
Thanks Sebastian

**Comment by syamsulalam** [comment]
i think it's not because raspberry pi too weak but simply serpbear has trouble with too much keyword. all are being requested at the same time but scraper can only handle so much at a time so it works for some but then many are not yet tracked and forgotten by serpbear.

**Comment by syamsulalam** [comment]
okay found the problem, in my case it is because i'm using free scraper from scrapingrobot which is very slow (it has only 1 concurrent connection at the same time) so for hundreds or even thousands of keyword it takes 30 sec approximately per keyword which is very very slow

**Comment by ingeniumdesign** [comment]
Hi and thanks , you edit this setting?

![Screenshot_2024-12-07-08-44-52-26_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/10eaaf1e-c277-4e0a-99e9-acc858b14f1d)

**Comment by syamsulalam** [comment]
yes you edit that settings, change to serper.dev (it has free 2500 credit) it will be a lot faster in scraping (10x faster or more).

**Comment by ingeniumdesign** [comment]
Perfect, and what is the best Scraping Frequency and Keyword Scrape Delay Settings ?

**Comment by syamsulalam** [comment]
> Perfect, and what is the best Scraping Frequency and Keyword Scrape Delay Settings ?

keyword scrape delay: 0
scraping frequency: i set it to monthly or weekly, no need to get day by day data because it can vary just because. as long as you get overall view of where your keyword is. also it won't burn credit as much.
